docs/src/whatsnew/latest.rst

```diff
@@ -24,7 +24,7 @@ This document explains the changes made to Iris for this release
 📢 Announcements
 ================
 
-#. N/A
+#. `lbdreyer`_ relicensed Iris from LGPL-3 to BSD-3. (:pull: `5577`)
```

Comment:
```suggestion
#. `@lbdreyer`_ relicensed Iris from LGPL-3 to BSD-3. (:pull: `5577`)
```

Reply:
Thanks for fixing @bjlittle! I don't think I've ever managed to get the formatting of a whatsnew entry correct the first time!!